.github/workflows/staging-scheduled-deploy.yml

```diff
@@ -2,7 +2,8 @@
 name: Scheduled Deploy From Main to Staging
 on:
   schedule:
-    - cron: '0 10 * * 1-5'
+    # Invoke every Mon-Sat
+    - cron: '0 10 * * 1-6'
```

Comment:
Have been meaning to add this, since previously, it will not detect any merges into main after friday, this will fix that and ensure no merges are missing over the weekend